### ISSUE TITLE
Create pipeline for retrieving annotated images as training data

### ISSUE THREAD

**Issue by ktl014** [hab-spc/hab-ml#41]
### Story
As an ML engineer, I want to be able to access annotated images to use as additional training data. I should also be clearly communicated and visually see what images were annotated to under a particular class.

### Definition of Done
- [ ] Screenshots of grabbing lab annotated images and creating a training/validation set form it.
- [ ] Code review of the lab image pipeline API.

### Subtasks
- [ ] Pipeline to grab annotated lab data
- [ ] Connected pipeline to create training/validation set

**Comment by ktl014** [comment]
Hey @igneel73, are we gonna be discussing the pipeline for grabbing annotated lab data during our 1:1 talk?
